docs/src/dsp.md

```diff
@@ -0,0 +1,12 @@
+# Signal Processing
+
+It is expected that these functions will be moved to a different package.
+
+```@docs
+FFTW.filt
+FFTW.filt!
+FFTW.deconv
+FFTW.conv
+FFTW.conv2
+FFTW.xcorr
```

Comment:
A natural place for this kind of stuff would be [`DSP.jl`](https://github.com/JuliaDSP/DSP.jl)